### ISSUE TITLE
Instance of 'std::length_error' of a genome

### ISSUE THREAD

**Issue by olu2016** [nf-core/sarek#963]
### Description of the bug

My latest Sarek run on normal-tumor paired samples for one patient using some tools (freebayes,mutect2,strelka,tiddit,cnvkit,controlfreec,merge) ended up with an instance of 'std::length_error' related to  Homo_sapiens_assembly38.fasta.bed.

### Command used and terminal output

```console
cmd:

nextflow run -resume /AnalysisFolder/WES/SAREK2/sarek/main.nf \
        -profile vm03 \
        --input /AnalysisFolder/WES/SAREK2/sarek/samplesheet.csv \
        --outdir /AnalysisFolder/WES/SAREK2/sarek_out \
        --tools freebayes,mutect2,strelka,tiddit,cnvkit,controlfreec,merge \
        -with-conda true

Terminal output:

..Reading Homo_sapiens_assembly38.fasta.bed
  ..Your file must be in .BED format, and it must be sorted
  Number of exons analysed in chromosome HLA-DRB1*15:02:01 : 1
  Average exon length in chromosome HLA-DRB1*15:02:01 : 10313
  ..Reading Homo_sapiens_assembly38.fasta.bed
  ..Your file must be in .BED format, and it must be sorted
  Number of exons analysed in chromosome HLA-DRB1*15:03:01:01 : 1
  Average exon length in chromosome HLA-DRB1*15:03:01:01 : 11567
  ..Reading Homo_sapiens_assembly38.fasta.bed
  ..Your file must be in .BED format, and it must be sorted
  Number of exons analysed in chromosome HLA-DRB1*15:03:01:02 : 1
  Average exon length in chromosome HLA-DRB1*15:03:01:02 : 11569
  ..Reading Homo_sapiens_assembly38.fasta.bed
  ..Your file must be in .BED format, and it must be sorted
  Number of exons analysed in chromosome HLA-DRB1*16:02:01 : 1
  Average exon length in chromosome HLA-DRB1*16:02:01 : 11005
  ..[genomecopynumber] Starting reading tumor_sample_vs_normal_sample.normal.mpileup.gz
  ..finished reading tumor_sample_vs_normal_sample.normal.mpileup.gz
  terminate called after throwing an instance of 'std::length_error'
    what():  basic_string::_M_create
  .command.sh: line 60: 64724 Aborted                 (core dumped) freec -conf config.txt

Work dir:
  /AnalysisFolder/WES/SAREK2/work/72/a5ddf99e016e01cf77585832091f59

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`

/AnalysisFolder/WES/SAREK2/work/72/a5ddf99e016e01cf77585832091f59/.command.run
```

### Relevant files

[.nextflow.log](https://github.com/nf-core/sarek/files/10930981/default.nextflow.log)

### System information

Local Ubuntu machine

**Comment by olu2016** [comment]
Here's vm03 profile:

`    vm03 {
        params.max_memory      = '120.GB'
        params.max_cpus        = 24
        params.max_time        = '72.h'
        params.enable_conda    = true
        env.conda              = "/AnalysisFolder/miniconda3/condabin/conda"
        conda.cacheDir         = "/AnalysisFolder/conda_envs"
        conda.createTimeout    = '1 h'
        charliecloud.enabled   = false
        docker.enabled         = false
        podman.enabled         = false
        shifter.enabled        = false
        singularity.enabled    = false
        params.genome          = 'GATK.GRCh38'
        params.igenomes_base   = '/AnalysisFolder/WES/genome'
        params.vep_cache       = "${params.igenomes_base}/.vep/"
        params.snpeff_cache    = "${params.igenomes_base}/SnpEff"
    }`

**Comment by olu2016** [comment]
vm03 {
        params.max_memory      = '120.GB'
        params.max_cpus        = 24
        params.max_time        = '72.h'
        params.enable_conda    = true
        env.conda              = "/AnalysisFolder/miniconda3/condabin/conda"
        conda.cacheDir         = "/AnalysisFolder/conda_envs"
        conda.createTimeout    = '1 h'
        charliecloud.enabled   = false
        docker.enabled         = false
        podman.enabled         = false
        shifter.enabled        = false
        singularity.enabled    = false
        params.genome          = 'GATK.GRCh38'
        params.igenomes_base   = '/AnalysisFolder/WES/genome'
        params.vep_cache       = "${params.igenomes_base}/.vep/"
        params.snpeff_cache    = "${params.igenomes_base}/SnpEff"
    }

**Comment by maxulysse** [comment]
So nothing special with intervals?
I'm wondering where this `Homo_sapiens_assembly38.fasta.bed` is coming from...
any idea @FriederikeHanssen

**Comment by FriederikeHanssen** [comment]
yeah me too. can you send the .command.log and .command.err from the failed process. If you navigate to the sub-work directory that is printed, you should be able to find it

**Comment by olu2016** [comment]
.command.log and .command.err attached. Thanks.
[commandDoTerrorLOG.zip](https://github.com/nf-core/sarek/files/10934668/commandDoTerrorLOG.zip)

**Comment by FriederikeHanssen** [comment]
No idea. Sorry :( Are you specifying the matching target bed file to your wes data somewhere? and myabe adding the flag `--wes` would help, it tunes some of the freec parameters recommended for exome data

**Comment by olu2016** [comment]
Thank you guys! I will try --wes and keep you posted.

**Comment by FriederikeHanssen** [comment]
Has this worked? If yes, could you close the issue?

**Comment by olu2016** [comment]
Apologies, I did not report back earlier. Just to confirm that the step worked. Thank you!
